manifests/init.pp

```diff
@@ -453,12 +453,8 @@
     }
   }
 
-  if defined('::firewall') {
-    if $configure_firewall == undef {
-      fail('The firewall module is included in your manifests, please configure $configure_firewall in the jenkins module')
-    } elsif $configure_firewall {
-      include jenkins::firewall
-    }
+  if defined('::firewall') and $configure_firewall {
```

Comment:
does this refer to a fact? What about `facts('firewall')` function?

Reply:
I actually don't know for sure, but I think whether https://github.com/puppetlabs/puppetlabs-firewall/blob/master/manifests/init.pp is included (before compiling this class).

Reply:
okay. In that case lets keep it like it is